qfieldsync/ui/cloud_login_dialog.ui

```diff
@@ -179,6 +179,41 @@
      </property>
     </widget>
    </item>
+   <item>
+    <widget class="QGroupBox" name="ssoLoginOngoingGroupBox">
+     <property name="title">
+      <string/>
+     </property>
+     <layout class="QFormLayout" name="formLayout_2">
+      <item row="0" column="0">
+       <widget class="QLabel" name="ssoSpinnerLabel">
+        <property name="maximumSize">
+         <size>
+          <width>36</width>
+          <height>36</height>
```

Comment:
This looks quite big on the screenshot, I'd match the height of the spinner with that of the cancel sign in button.